examples/stable-diffusion/run_1x.sh

```diff
@@ -23,4 +23,6 @@ python train_text_to_image_sdxl.py \
   --validation_prompt="a robotic cat with wings" \
   --validation_epochs 48 \
   --checkpointing_steps 2500 \
-  --logging_step 10 2>&1 | tee log_1x_r512.txt
+  --logging_step 10 \
```

Comment:
can you use a different script? like run_1x_det.sh? since we will release run_1x.sh, and use run_1x_det for acc

Reply:
ok.